### PR TITLE
feat(translate): translate package items

### PR DESCRIPTION
**Pull request by petrjasek**
when translating a package translate each item contained in it.

SDFID-270

### CODE REVIEW

File: features/content_translate.feature [hunk 1]
```diff
@@ -66,3 +66,54 @@ Feature: Translate Content
...
+      Then we get new resource
+      When we post to "/archive/translate"
+      """
+      {"guid": "package", "language": "de", "desk": "#desks._id#"}
+      """
+  
+      When we get "/archive/#translate._id#"
+      Then we get existing resource
+      """
+      {"translated_from": "package", "operation": "translate", "task": {"stage": "#desks.working_stage#"}}
```
Comment (ioanpocol) [inline features/content_translate.feature]: How do we know that the new items are really part of the new package?

Comment (petrjasek) [reply features/content_translate.feature]: I didn't know how to test this because those new item ids are generated, any suggestions?

Comment (ioanpocol) [reply features/content_translate.feature]: You can set for  "residRef" the value "__any_value__", and we at least know that there are 2 items in package.

Comment (petrjasek) [reply features/content_translate.feature]: added a new step definition to check package items

**Pull request merged**